gateway-provider-identity-assertion-no-doas/src/main/java/org/apache/knox/gateway/identityasserter/filter/IdentityAsserterNoImpersonationHttpServletRequestWrapper.java

```diff
@@ -25,10 +25,10 @@
 import java.util.List;
 import java.util.Map;
 
-public class IdentityAsserterNoDoAsHttpServletRequestWrapper extends
+public class IdentityAsserterNoImpersonationHttpServletRequestWrapper extends
```

Comment:
Not critical, but following my other comments around naming, this one could rather be something like `NoImpersonationIdentityAsserterHttpServletRequestWrapper` or `NoImpersonationAsserterRequestWrapper`

Reply:
Ahh sure!